bundles/org.eclipse.passage.lic.base/src/org/eclipse/passage/lic/internal/base/permission/observatory/CheckSchedule.java

```diff
@@ -0,0 +1,82 @@
+/*******************************************************************************
+ * Copyright (c) 2019 ArSysOp
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     ArSysOp - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.passage.lic.internal.base.permission.observatory;
+
+import java.time.temporal.ChronoUnit;
+
+/**
+ * <p>
+ * Schedule for observatory checks.
+ * </p>
+ * 
+ * <p>
+ * Been defined for <i>5 minutes</i>, causes the observatory to check for new
+ * expired entries each 5 minutes (or so).
+ * </p>
+ *
+ * @since 0.6.1
+ */
+public class CheckSchedule {
+
+	private final int amount;
+	private final ChronoUnit unit;
+
+	/**
+	 * <p>
+	 * Use desired units to define the most suitable check schedule.
+	 * </p>
+	 * 
+	 * <p>
+	 * For example, {@code new CheckSchedule(2, ChronoUnit.DAYS)}, been given to an
+	 * observatory, will cause the closest expiration check to happen only in two
+	 * days since the observatory opening.
+	 * </p>
+	 * 
+	 * @param amount number of units
+	 * @unit {@linkplain ChronoUnit} constant to measure {@code amount}
+	 * @since 0.6.1
+	 */
+	public CheckSchedule(int amount, ChronoUnit unit) {
+		this.amount = amount;
+		this.unit = unit;
+	}
+
+	/**
+	 * Default schedules are measured in minutes.
+	 * 
+	 * @param minutes number of minutes
+	 * @since 0.6.1
+	 */
+	public CheckSchedule(int minutes) {
+		this(minutes, ChronoUnit.MINUTES);
+	}
+
+	/**
+	 * Default schedule is 10 minutes.
+	 * 
+	 * @since 0.6.1
```

Comment:
Normally "since" tag value has only major.minor form

Reply:
Does this mean, it can be not equal to current bundle version? Must simple `0.6` appear there?

Reply:
Exactly. API may not be added with service release, only with minor one. So, the "0.6.1" is "0.6"

Reply:
Does this mean we must mention since-version only on public apis? (which is not ther case here)